<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBOX</class>
 <widget class="QMainWindow" name="DBOX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBOX</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="dbox_tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="preprocessor">
       <attribute name="title">
        <string>Preprocessor</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QToolBox" name="preprocessor_toolBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="source">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>200</width>
             <height>549</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Source</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTreeView" name="preprocessor_treeView_source">
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="settings">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>200</width>
             <height>549</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Settings</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="view">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>200</width>
             <height>549</height>
            </rect>
           </property>
           <attribute name="label">
            <string>View</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="preprocessor_horizontalSpacer_1">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QRadioButton" name="preprocessor_radioButton_compareView">
                <property name="text">
                 <string>Compare</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="preprocessor_radioButton_compactView">
                <property name="text">
                 <string>Compact</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="preprocessor_horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="preprocessor_verticalSpacer_1">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>360</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QWidget" name="preprocessor_widget_compareView" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QComboBox" name="preprocessor_comboBox_compare1">
                   <property name="maximumSize">
                    <size>
                     <width>200</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGraphicsView" name="preprocessor_graphicsView_compare1"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QComboBox" name="preprocessor_comboBox_compare2">
                   <property name="maximumSize">
                    <size>
                     <width>200</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGraphicsView" name="preprocessor_graphicsView_compare2"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="preprocessor_widget_compactView" native="true">
              <layout class="QGridLayout" name="gridLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QLabel" name="preprocessor_label_original">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Original</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="preprocessor_horizontalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="preprocessor_checkBox_contrast">
                     <property name="text">
                      <string>Contrast Enhanced</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QGraphicsView" name="preprocessor_graphicsView_original"/>
                 </item>
                </layout>
               </item>
               <item row="0" column="1">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="preprocessor_label_mask">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Mask</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGraphicsView" name="preprocessor_graphicsView_mask"/>
                 </item>
                </layout>
               </item>
               <item row="0" column="2">
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="preprocessor_label_map">
                     <property name="text">
                      <string>Map</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QComboBox" name="preprocessor_comboBox_map">
                     <property name="minimumSize">
                      <size>
                       <width>120</width>
                       <height>0</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QGraphicsView" name="preprocessor_graphicsView_map"/>
                 </item>
                </layout>
               </item>
               <item row="1" column="0" colspan="3">
                <spacer name="preprocessor_verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>600</width>
                   <height>13</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="2" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QLabel" name="preprocessor_label_enhanced">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Enhanced</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGraphicsView" name="preprocessor_graphicsView_enhanced"/>
                 </item>
                </layout>
               </item>
               <item row="2" column="1">
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <widget class="QLabel" name="preprocessor_label_binarized">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Binarized</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGraphicsView" name="preprocessor_graphicsView_binarized"/>
                 </item>
                </layout>
               </item>
               <item row="2" column="2">
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="preprocessor_label_skeleton">
                     <property name="text">
                      <string>Skeleton</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="preprocessor_horizontalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="preprocessor_checkBox_invertedSkeleton">
                     <property name="text">
                      <string>Inverted</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QGraphicsView" name="preprocessor_graphicsView_skeleton"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="preprocessor_progressBar">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1179</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
